static/js/reconciliation.js

```diff
@@ -26,7 +26,8 @@
             state = operations[state]();
         }).appendTo($rec);
 
-        var $1 = $rec.find('td:contains("Invoice 1"), td:contains("Payment 1")')
+        var $1 = $rec.find('td:contains("Invoice 1"), td:contains("Partial payment 1/2"),
+            td:contains("Partial payment 2/2")')
```

Comment:
This will conflict with the forward-port of https://github.com/odoo/documentation/pull/6896